.travis.yml

```diff
@@ -57,6 +59,10 @@ after_success:
       docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
       docker push $DOCKER_IMAGE:$DOCKER_TAG;
     fi
+  - if [ "$DOCKER_TAG" == "master" ]; then
```

Comment:
indeed, it is just a duplicate tag. But since "latest" is a special tag in docker terminology, omitting it would cost us. The best option I've found was to make it equivalent to master. Other ideas?